package.json

```diff
@@ -85,6 +86,7 @@
     "yjs": "13.6.27"
   },
   "dependencies": {
+    "@wordpress/private-apis": "^1.31.0",
```

Comment:
Where is this used?

Reply:
Good catch. From earlier attempts. Removed in c65b789